src/Guide/Api/Types.hs

```diff
@@ -134,7 +135,28 @@ data ItemSite route = ItemSite
 
 -- | Working with item traits
 data TraitSite route = TraitSite
-  { _deleteTrait :: route :-
+  { _createTrait :: route :-
+      Summary "Create a new trait in the given item"
+      :> Description "Returns the ID of the created trait."
+      :> ErrorResponse 400 "'text' not provided"
+      :> "item"
+      :> Capture "item" (Uid Item)
+      :> "trait"
+      :> Capture "type" TraitType
+      :> QueryParam' '[Required, Strict] "text" Text
+      :> Post '[JSON] (Uid Trait)
+
+  , _setTrait :: route :-
+      Summary "Update a trait in the given item"
+      :> Description "Returns nothing"
```

Comment:
Well, this description is not useful because it would be very clear from the schema, so it's better to remove it.